WalkingChallenge/Notifications/NotificationTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NotificationTableViewCell.swift
+//  WalkingChallenge
+//
+//  Created by Husein Kareem on 4/12/19.
+//  Copyright © 2019 AKDN. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class NotificationTableViewCell: UITableViewCell {
+
+  static let CellIdentifier: String = "NotificationCell"
+
+  lazy var notificationInfoLabel: UILabel = UILabel()
+  lazy var notificationDateLabel: UILabel = UILabel()
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.backgroundColor = Style.Colors.DeepTurquoise.withAlphaComponent(0.0735)
```

Comment:
Where was this color selected?  We should be trying to stick to the color style guide

Reply:
It's from the Marvel design.

Reply:
I don't see turquoise in the mockup.  Got a link?